### PR TITLE
Native HCG support implementation

### PR DESCRIPTION
**Pull request by acterhd**
I added support of [HCG](https://github.com/acterhd/hcg-color) color model. 

Plug-in official for HCG.
https://github.com/acterhd/less-hcg

### CODE REVIEW

**Comment by seven-phases-max** [comment]
It seems you've ignored #2803, did you? I So technically it's not that I specifically would argue, but I actually wonder if this is worth being in the core instead of a plugin.

File: lib/less/tree/color.js [hunk 1]
```diff
@@ -132,6 +132,28 @@ Color.prototype.toHSL = function () {
     }
     return { h: h * 360, s: s, l: l, a: a };
 };
+Color.prototype.toHCG = function () {
```
Comment (seven-phases-max) [inline lib/less/tree/color.js]: This function does not seems to be used anywhere so what is it its purpose?

Comment (acterhd) [inline lib/less/tree/color.js]: For HCG planned modifiers (for example "purity"). But I thinking about names, that will not conflict with HSL/HSV/HWB.

**Comment by seven-phases-max** [comment]
So I'm adding "needs decision" flag and referencing #1853 once again. While two functions is probably is not something to bother arguing for/against with, further this leads to all that sort of proposals/extensions discussed in thus it's better to decide earlier (plugin? plugin?).

**Comment by acterhd** [comment]
Okay. I'm not tested, but here is Less plug-in. This is temporary solution. 
https://github.com/acterhd/hcg-color/tree/master/less

I understand - you have too much pull requests?

**Pull request closed without merging**